avs_rvtools_analyzer/services/azure_openai_service.py

```diff
@@ -0,0 +1,288 @@
+        if self.configured_via_env:
+            logger.warning("Azure OpenAI is already configured via environment variables")
+            return True
+            
+        try:
+            self.client = AzureOpenAI(
+                azure_endpoint=azure_endpoint,
+                api_key=api_key,
+                api_version=api_version
+            )
+            self.is_configured = True
+            logger.info("Azure OpenAI client configured successfully")
+            return True
+        except Exception as e:
+            logger.error(f"Failed to configure Azure OpenAI client: {e}")
+            self.is_configured = False
+            return False
+
+    def get_risk_analysis_suggestion(
+        self, 
+        risk_name: str, 
+        risk_description: str, 
+        risk_data: List[Dict[str, Any]], 
+        risk_level: str,
+        deployment_name: str = "gpt-4"
+    ) -> Optional[str]:
+        """
+        Get AI-powered suggestions for a specific risk.
+        
+        Args:
+            risk_name: Name of the risk (e.g., "detect_vusb_devices")
+            risk_description: Description of the risk
+            risk_data: List of risk data items
+            risk_level: Risk severity level
+            deployment_name: Azure OpenAI deployment name (ignored if configured via env vars)
+            
+        Returns:
+            str: AI-generated suggestion or None if failed
+        """
+        if not self.is_configured or not self.client:
+            logger.warning("Azure OpenAI client not configured")
+            return None
+
+        # Use env deployment name if configured via env vars, otherwise use provided one
+        model_name = self.deployment_name if self.configured_via_env else deployment_name
+
+        try:
+            # Prepare the prompt
+            prompt = self._build_risk_analysis_prompt(
+                risk_name, risk_description, risk_data, risk_level
+            )
+            
+            # Make the API call
+            response = self.client.chat.completions.create(
+                model=model_name,
+                messages=[
+                    {
+                        "role": "system",
+                        "content": "You are an expert Azure VMware Solution migration consultant. Provide practical, actionable recommendations for addressing migration risks."
+                    },
+                    {
+                        "role": "user",
+                        "content": prompt
+                    }
+                ],
+                max_tokens=1000,
+                temperature=0.3
+            )
+            
+            suggestion = response.choices[0].message.content
+            logger.info(f"Generated suggestion for risk: {risk_name}")
+            return suggestion
+            
+        except Exception as e:
+            logger.error(f"Failed to get AI suggestion for {risk_name}: {e}")
+            return None
+
+    def _build_risk_analysis_prompt(
```

Comment:
Seems that risk data are not sent properly to the AI model that is always mentioning that the risk do not apply to the current situation when answering. Ensure that test data is well sent to the model for analysis.